coopterrones/Overlook
# Feature/css styling

# Pull Request Template:


#### What's this PR do?
1. styling update with neumorphism
#### Where should the reviewer start?
*index.html
*base.scss
#### How should this be manually tested?
*check styling 
*make sure no issues or errors in console on load 
*check for layouts matching in manager / customer view 
#### Any background context you want to provide?
*Not responsive yet 
*Needs layout change to avail rooms area 
*dark mode hopefully
